src/main/java/cz/brmlab/yodaqa/pipeline/structured/StructuredPrimarySearch.java

```diff
@@ -190,6 +205,9 @@ protected void propertyToAnswer(JCas jcas, PropertyValue property,
 		}
 
 		ai.addToIndexes();
+
+		createPropertyLabels(questionView,property,jcas);
+
```

Comment:
Are the blank lines meaningful? (Esp. the one after.)

If we are creating some property labels as a potential side-effect, we should document that in the class description, incl. the system property name.